wtxmgr/example_test.go

```diff
@@ -155,7 +155,12 @@ func Example_basicUsage() {
 	}
 
 	// Create (or open) the transaction store in the provided namespace.
-	s, err := wtxmgr.Create(ns)
+	err = wtxmgr.Create(ns)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	s, err := wtxmgr.Open(ns)
```

Comment:
This is another example of why Create should just return the manager like it used to.  Putting more burden on the caller like this doesn't have any benefit.